readme.markdown

```diff
@@ -480,6 +492,11 @@ For a package named "foobar", the recommended project structure is the following
     ├── nim.cfg
     ├── tfoo1.nim   # First test
     └── tfoo2.nim   # Second test
+└── samples
+    ├── demo1       # Demo application
+    ├── simpleapp   # Application template
+    └── foo
+        └── subfoo  # can be nested
```

Comment:
The lack of file extension here implies that these are all binaries

Reply:
these are subfolders.
(eg: multithreading/channels; multithreading/threadpool )

the idea was:
for a framework sometimes directory structures needed with files in them.
polluting project dir with include files is not pretty, include files can go into subfolder,
(eg: input handling: case event of: "keyUp:")
while main.nim goes into project folder - as it starts the show (app.run).

structure extended with files:
    ├── demo1       # Demo application
     |    └── demo.nim
     |  
    ├── simpleapp   # Application template
     |    └── main.nim
     |  
    └── multithreading
          └── channels  
           |    └── demo.nim
          └── threadpool
                └── demo.nim